drivers/filesystems/cdfs/volinfo.c

```diff
@@ -160,6 +160,8 @@ Return Value:
         //  and false if it couldn't wait for any I/O to complete.
         //
 
+        RtlZeroMemory(Irp->AssociatedIrp.SystemBuffer, Length);
```

Comment:
Hm, does it mean Windows also has this issue? :)

Reply:
Which issue? Structure alignment or using RtlZeroMemory to clear memory?

Reply:
I mean cdfs driver is taken from Microsoft samples repository

Reply:
No, I haven't. I don't know cdfs driver is taken from Microsoft.

Reply:
So please create PR for MS repo https://github.com/microsoft/Windows-driver-samples

Reply:
I created a pull request to Microsoft repository: https://github.com/microsoft/Windows-driver-samples/pull/536